internal/view/pod.go

```diff
@@ -337,6 +356,25 @@ func attachIn(a *App, path, co string) {
 	}
 }
 
+func computeDebugArgs(path, co string, kcfg *string, os string, image string) []string {
+	args := make([]string, 0, 15)
+	args = append(args, "debug", "-it")
+
+	ns, po := client.Namespaced(path)
+	if ns != client.AllNamespaces {
+		args = append(args, "-n", ns)
+	}
+	args = append(args, po)
+	if kcfg != nil && *kcfg != "" {
+		args = append(args, "--kubeconfig", *kcfg)
+	}
+
+	args = append(args, "--target", co)
```

Comment:
`target` and `image` have the same syntax, but in one case you use `=` and don't use an equal sign in another. Why?

Reply:
Nice catch. Copy paste miss I think. I'll remove the `=`